am-lib/src/main/java/uk/gov/hmcts/reform/amlib/enums/Permission.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.reform.amlib.enums;
+
+/**
+ * Exposes a set of enum values used to set permissions for Access Management.
+ * Each of the values is a power of two. The reason for that is that in Access Management
+ * there might be multiple permissions that need to be assigned: ie. READ + CREATE + HIDE.
```

Comment:
HIDE appears in Javadoc while it does not exist any longer. When code is removed please check documentation as well.

Reply:
👍